### ISSUE TITLE
Review keyboard navigation behaviour for removing chips in Suggestion component

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystem#4748]
Keyboard navigation for removing chips in the Suggestion component technically works, but the current behaviour may not align with user expectations.

Currently, users need to press the left arrow key to move focus to the chips. While functional, this may feel unintuitive, as users might expect to use Tab to navigate between interactive elements.

We should review whether the current arrow key behaviour is the most appropriate interaction pattern and consider if using Tab navigation, or another approach, would provide a more intuitive and accessible experience.

**Comment by mrosvik** [comment]
Any thoughts about this @eirikbacker ?

**Comment by eirikbacker** [comment]
Hi @mrosvik! 🌟 Thanks for good feedback! ☺️ 
You say "the current behavoiur may not align with user expectations" - I presume this comes from some user feedback or testing? ☺️

The current keyboard navigation is intentional:
1. as the Suggestion element should be considered a single form-element, the keyboard navigation should also reflect this as a single tab-stop. This aligns with native elements such as `<select>`, `<datalist>` or `<input type="date">` - which also are single form-elements, but consists of several interactive pieces.
2. other elements - such as `tabs` - might also seem like good candidates for several tab-stops, but the W3C ARIA-specifications instead removing several elements from the tab sequence, and rather using a [Roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex) solution for quicker navigation around the page.
3. as a true multi select combobox W3C-specification does not currently exist, we have aligned the implementation with how native operative systems solve this patten - as this is the most familiar pattern for users. Multi select comboboxes are often found in search interfaces, or subject-lines in mail-programs (also actually in gmail.com). Here, keyboard arrows are also used to navigate between chips. 

This said, we welcome feedback to the u-elements-project, and would like to know more about the basis of this feedback 🤗
